R/plot_allcurves.R

```diff
@@ -0,0 +1,86 @@
+#' Plot All Fitted Curves From tcplfit2_core
+#'
+#' This function takes in output from `tcplfit2_core` and plots
+#' the observed concentration response data with all fitted curves.
+#' The function returns a ggplot object that is capable of
+#' taking additional layers of ggplot add-ons.
+#'
+#' @param output Output from `concRespCore` or `tcplhit2_core`, containing information
+#' about the winning curve fit of a compound.
+#'
+#' @param conc Vector of concentrations (not in log units).
+#'
+#' @param resp Vector of corresponding responses.
```

Comment:
Suggest aligning the definitions of "conc" and "resp" with the tcplfit2_core parameter definitions.